doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -10,6 +10,29 @@
         "#1": "Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For an SQLite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
         "#3": "Configure your db driver and server_version in config/packages/doctrine.yaml",
-        "DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name"
+        "DATABASE_URL": "postgres://symfony:!ChangeMe!@db/symfony?serverVersion=11"
+    },
+    "dockerfile": [
+        "RUN apk add --no-cache libpq postgresql-dev && \\",
+        "\tdocker-php-ext-install pdo pdo_pgsql && \\",
+        "\tapk del postgresql-dev"
+    ],
+    "docker-compose": {
+        "services": [
+            "db:",
+            "  image: postgres:11-alpine",
+            "  environment:",
+            "    - POSTGRES_DB=symfony",
+            "    - POSTGRES_USER=symfony",
+            "    # You should definitely change the password in production",
+            "    - POSTGRES_PASSWORD=!ChangeMe!",
+            "  volumes:",
+            "    - db-data:/var/lib/postgresql/data:rw",
+            "    # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!",
+            "    # - ./docker/db/data:/var/lib/postgresql/data:rw",
+            "  ports:",
+            "    - \"5432:5432\""
```

Comment:
Like for the Messenger recipe, I would let Docker choose a random port for localhost to allow running several projects in parallel.